### PR TITLE
IG Results

### PR DESCRIPTION
**Pull request by saquino0827**
# Implementation Guide Results Flow

Updates the IG to add the results flow and ORU type documentation

## Issue

Add a link to the issue here.  Consider using
[closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
if the this PR isn't for a story (stories will be closed through different means).

## Checklist

- [ ] I have added tests to cover my changes
- [ ] I have added logging where useful (with appropriate log level)
- [ ] I have added JavaDocs where required
- [ ] I have updated the documentation accordingly

**Note**: You may remove items that are not applicable

### CODE REVIEW

File: ig/input/fsh/logical-oru-01.fsh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// defines the ORU-R01 (loosely) input that is expected for newborn screening
+
+Logical: ORUR01
+Id: oru-R01-logical-model
+Title: "ORU-R01"
+Description: "The expected input and output for ORU formatted data for the newborn screening ETOR workflow"
+* MSH 1..1 MSHSegment "MSH segment"
+* PID 1..1 PIDSegment "PID segment"
+* NK1 1..1 NK1Segment "NK1 segment"
```
[the remark this answers is not in the document]
Comment (saquino0827) [reply ig/input/fsh/logical-oru-01.fsh:9]: K, I was going off of what we had for OML but that makes more sense. I'll see if I can find documentation on that quickly and if not, i'll reach out to John

File: ig/input/fsh/logical-oru-01.fsh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* ORC 1..* ORCSegment "ORC segment"
+* OBR 1..* OBRSegment "OBR segment"
+* SPM 1..1 SPMSegment "SPM segment"
```
[the remark this answers is not in the document]
Comment (saquino0827) [reply ig/input/fsh/logical-oru-01.fsh:12]: Same with OML but I checked https://hl7-definition.caristix.com/v2/HL7v2.5.1/TriggerEvents/ORU_R01 and it says the SPM segment contains 1..1 SPM and 1..* OBX. But I will reach out to John about it.

[the remark this answers is not in the document]
Comment (basiliskus) [reply ig/input/fsh/logical-oru-01.fsh:12]: I also though `SPM` was not repeatable but checking the caristix docs the `SPECIMEN` section is repeatable. But I see that `SPM` is not, so I'm not sure

File: ig/input/fsh/logical-oru-01.fsh (hunk 1 above)
[the remark this answers is not in the document]
Comment (JohnNKing) [reply ig/input/fsh/logical-oru-01.fsh:9]: Technically, I think NK1 is 0..* (https://hl7-definition.caristix.com/v2/HL7v2.5.1/TriggerEvents/ORU_R01).

However, I'd expect at least one for the Mother's information, unless:
1. The ORU doesn't echo back what was in the OML for NK1
2. or the Mother's information was lacking from the OML to begin with

[the remark this answers is not in the document]
Comment (saquino0827) [reply ig/input/fsh/logical-oru-01.fsh]: Updated NK1 to 0..*

File: ig/input/fsh/logical-oru-01.fsh (hunk 2 above)
[the remark this answers is not in the document]
Comment (saquino0827) [reply ig/input/fsh/logical-oru-01.fsh]: I updated SPM to 0..1 based on the examples provided here.

**Review by basiliskus — APPROVED** [review]

**Review by jorg3lopez — APPROVED** [review]
Great job!

**Pull request merged**